setup.py

```diff
@@ -64,6 +63,12 @@ def read_file(filename, encoding='utf8'):
 else:
     install_requires.append('mock')
 
+# Debian squeeze support, cf. #280
+if sys.version_info[0] == 2:
```

Comment:
Just noting this now for when this is given a full review, but my understanding is changes like this will require us to build multiple wheels of `certbot` when pushing the packages to PyPI. We'll need a Python 2 wheel and a Python 3 wheel.

I just encountered a similar problem [here](https://github.com/certbot/certbot/pull/2061#discussion_r71929407) but luckily there was a workaround. I'm not aware of a single `dialog` package we can use here so this may be inevitable (unless we want to write our own `dialog` compatibility library or drop support for `dialog`, although at that point making and testing two wheels is probably easier) but I wanted to make sure we kept this in mind before merging.